plugins/git-release-manager/src/features/Stats/Row/Row.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { DateTime } from 'luxon';
+import {
+  Collapse,
+  IconButton,
+  Link,
+  makeStyles,
+  TableCell,
+  TableRow,
+} from '@material-ui/core';
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp';
+
+import { ReleaseStats } from '../contexts/ReleaseStatsContext';
+import { RowCollapsed } from './RowCollapsed/RowCollapsed';
+
+const useRowStyles = makeStyles({
+  root: {
+    '& > *': {
+      borderBottom: 'unset',
+    },
+  },
+});
+
+interface RowProps {
+  baseVersion: string;
+  releaseStat: ReleaseStats['releases']['0'];
+}
+
+export function Row({ baseVersion, releaseStat }: RowProps) {
+  const [open, setOpen] = useState(false);
+  const classes = useRowStyles();
+
+  return (
+    <>
+      <TableRow className={classes.root}>
+        <TableCell>
+          <IconButton
+            aria-label="expand row"
+            size="small"
+            onClick={() => setOpen(!open)}
+          >
+            {open ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
```

Comment:
i think having closed being a chevron pointing to the right (not up) is kinda standard (that's what github does right here in my review flow!)